src/NuGet.Clients/VsExtension/VsCredentialProviderImporter.cs

```diff
@@ -76,6 +76,11 @@ public class VsCredentialProviderImporter
             _initializer = initializer ?? Initialize;
         }
 
+        // The export from TeamExplorer uses a named contract, so we need to import this one separately.
+        [Import("VisualStudioAccountProvider", typeof(IVsCredentialProvider), AllowDefault = true)]
+        public IVsCredentialProvider VisualStudioAccountProvider { get; set; }
```

Comment:
I think this would be safer as ImportMany, now that multiple providers are supported.  With a single import, if two extensions export the same named contract, neither will be imported.  With ImportMany, the import cannot be blocked in this way.